subscription-service/src/test/kotlin/com/egm/stellio/subscription/service/SubscriptionServiceTests.kt

```diff
@@ -899,4 +901,54 @@ class SubscriptionServiceTests : WithTimescaleContainer {
 
         assertEquals(res, true)
     }
+
+    /**
+     * Test for function 'getAllSubscriptions' with two subscription.
+     * One with the 'lastNotification' attribute set to null and the other with the 'lastNotification' now and
+     * 'timeInterval' set to 1
+     *
+     * @return list of subscription
+     */
```

Comment:
hmm...

Reply:
Maybe One with the 'lastNotification' attribute set to null and the other with the 'lastNotification' with instant date?